<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qqSetUp</class>
 <widget class="QWidget" name="qqSetUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>637</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>系统设置</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SettingPanel" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="KISetModelView"/>
     </item>
     <item>
      <widget class="QTableView" name="KOSetModelView"/>
     </item>
     <item>
      <widget class="QTableView" name="faultSetModelView"/>
     </item>
     <item>
      <widget class="QTableView" name="lineSetModelView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingPanel</class>
   <extends>QWidget</extends>
   <header location="global">.\Public\ModelView\settingpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
